app/docker/views/containers/console/containerConsoleController.js

```diff
@@ -121,7 +121,7 @@ angular.module('portainer.docker').controller('ContainerConsoleController', [
               .map((k) => k + '=' + params[k])
               .join('&');
 
-          initTerm(url, ExecService.resizeTTY.bind(this, params.id));
+          initTerm(url, ExecService.resizeTTY.bind(this, endpoint.Id, params.id), isLinuxTerminalCommand(execConfig.Cmd[0]));
```

Comment:
Make sure execConfig.Cmd[0] exists before using as an arg - it could be possible for commandStringToArray("") to give an array with no items

Reply:
This is still relevant

Reply:
After some testing, I've determined that this is absolutely fine.  An empty array still returns the correct result.

Reply:
Since execConfig.Cmd[0] can have the value undefined,
`isLinuxTerminalCommand(undefined)` is what to test for. This seems to be ok as long as commandStringToArray always returns an array. 

It's still a nice check since commandStringToArray may change in future to return undefined and getting execConfig.Cmd[0] of undefined would give a "Cannot read properties of undefined" error